src/modules/orders/orders.module.ts

```
@@ -7,10 +7,16 @@ import { OrderController } from './controllers/order.controller';
 import { OrderItem } from './entities/order-item.entity';
 import { ProductsModule } from '../products/products.module';
 import { Payment } from './entities/payment.entity';
+import { StatusService } from './services/status.service';
```

ESLint

[error] 'StatusService' is defined but never used.

(@typescript-eslint/no-unused-vars)